### ISSUE TITLE
unable to generate linter docs

### ISSUE THREAD

**Issue by pq** [dart-lang/linter#1091]
Attempting to re-generate the docs,

```dart tool/doc.dart -o ../io_linter/lints/```

errors:

```
file:///Users/pquitslund/.pub-cache/hosted/pub.dartlang.org/markdown-2.0.0/lib/src/util.dart:6:37: Error: Getter not found: 'ELEMENT'.
    const HtmlEscape(HtmlEscapeMode.ELEMENT).convert(html);
```

A `pub upgrade` warns:

```
Failed to precompile dart_style:format:
file:///Users/pquitslund/.pub-cache/hosted/pub.dartlang.org/analyzer-0.32.3/lib/src/dart/analysis/library_context.dart:96:34: Error: Can't return a value from a void function.
        return appendLibraryFiles(targetLibrary);
                                 ^
Failed to precompile test:test:
file:///Users/pquitslund/.pub-cache/hosted/pub.dartlang.org/analyzer-0.32.3/lib/src/dart/analysis/library_context.dart:96:34: Error: Can't return a value from a void function.
        return appendLibraryFiles(targetLibrary);
```

and a fresh attempt to generate docs now says:

```
[~/src/repos/linter] (master) $ dart tool/doc.dart -o ../io_linter/lints/
file:///Users/pquitslund/.pub-cache/hosted/pub.dartlang.org/analyzer-0.32.3/lib/src/dart/analysis/library_context.dart:96:34: Error: Can't return a value from a void function.
        return appendLibraryFiles(targetLibrary);
```

**Comment by pq** [comment]
> Pub 2.0.0-edge.b9a73c1fc61de71d01774411059d84a675d4c3f3

(rebuilding SDK now 🤞)

**Comment by pq** [comment]
Regarding the initial issue that stemmed (I think from a stale `markdown`), `pub upgrade` grabbed `2.0.2` so I think that's possibly resolved.

**Comment by pq** [comment]
Ah!  Looks like I'm hitting the same issue reported in https://github.com/dart-lang/sdk/issues/33901.

**Comment by cvanarsdale** [comment]
@pq This is also the same for me. Used the migration guide about 2 hours ago to get dart2 and I am unable to run webdev build or webdev serve. Having the same issue as kevmoo over at [https://github.com/dart-lang/sdk/issues/33901](url).

**Comment by pq** [comment]
@cvanarsdale : happy to say this should be fixed w/ https://github.com/dart-lang/sdk/commit/b32277988f5bb56bf94d2f107deb92a0fdff22b8 (and subsequent publish by @kevmoo)! 👍
